add more infos in PluginLicenseList

tinect opened the pull request:
### 1. Why is this change necessary?
There should be more information of current plugin-status in licenseList.

### 2. What does this change do, exactly?
Add some related info labels.
- check and add label if plugin is installed without update
- check and add label if plugin cannot be installed + tooltip-message
![image](https://user-images.githubusercontent.com/135993/89341356-776e0880-d6a1-11ea-9cb0-7a44d7d83fc3.png)


### 3. Describe each step to reproduce the issue or behaviour.


### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

janbuecker commented:
The container build should be as fast as possible with no more queries than needed. What is your use case for the change?

shyim commented:
Adding subscribers on some plugin configuration conditions or just easy access to plugin config without reading it on any subscriber again.. (if configs are not per subshop)

janbuecker commented:
We've discussed this change internally again and have decided to decline this pull request as we do not want to provide the database connection while the container is being build. This process should be as fast as possible.

Thanks @shyim

tinect committed:
add more infos in PluginLicenseList

tinect commented:
Rebased.
Fixed indentSize of https://github.com/shopware/platform/commit/e7f8829bad7d0daa3668896ac825b2e0932089e1#diff-c525b13e4a5b4bfe06758422b8379e86

philipreinken commented:
Hey there @tinect , thank you for putting in the time to create this pull request! Since it contains a feature change, we'll have to assess, whether or how this can be integrated. Please be aware that it might take a while before we can get back to you with a decision. I've also forwarded this to our design team and will leave this PR open in the meantime :slightly_smiling_face:
